### DISCUSSION TITLE
The editor timeline always display to-be-added objects as a new combo

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#23532] [Q&A] [answered]
As the title says, no matter if the "new combo" toggle is activated. The display is correct after the object is actually placed down. This is a bit annoying because I am often confused by whether I need to press Q :)
![2023-05-13 22 15 26](https://github.com/ppy/osu/assets/33959010/be5bbe24-77b1-473a-87d1-be8926b02b71)

**Comment by bdach** [comment]
There are two parts to this:

- The new object is always using the first combo colour, and a combo index of 1 (definite bug)
- The object being placed has a purplish tint applied to it, to distinguish it from the other timeline objects. This may give off an impression of the colour being different than it should be. Not sure how to solve this UX-wise, maybe opacity would be better used here, or some other sort of marking.

**Reply by bdach** [reply, marked as answer]
Escalated to issue https://github.com/ppy/osu/issues/23533

**Reply by Gyoshi** [reply]
On the playfield, the to-be-placed object has the same yellow circle as selected objects do. And it deselects currently selected objects when you prepare a hitcircle/slider to be placed.

The to-be-placed object can simply have the appearance of a selected object on the timeline too.
